is-the-average-whole/main.js

```diff
@@ -0,0 +1,20 @@
+function isTheAverageWhole(numbersArray)  {
+  var total = 0;
+  for(var i = 0; i < numbersArray.length; i++)  {
+    total += numbersArray[i];
+  }
+  var average = total / numbersArray.length;
+  // console.log('average of array:', average)
+  var stringNum = average.toString();
+  for(var j = stringNum.length - 1; j > 0; j--) {
+    if(stringNum[j].includes('.'))  {
```

Comment:
When you are going to use the includes method you don't need a loop. the includes method can iterate through the string on its own